bftengine/src/bftengine/Crypto.hpp

```diff
@@ -27,6 +27,7 @@ using std::string;
 
 namespace bftEngine {
 namespace impl {
+enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
```

Comment:
Do we still need it here?

Reply:
just in the meantime, until we remove Crypto completely